docs/DeveloperGuide.adoc

```diff
@@ -1226,6 +1265,75 @@ _{ more test cases ... }_
 
 _{ more test cases ... }_
 
+=== Return a loan functionality
+
+Prerequisites: List all loans using the list command. Ensure there are multiple loans in the list.
+
+. Returning a valid loan
+
+.. Choose a loan that has not been returned. These loans are marked with the ONGOING tag. Find the index number of it and run `return i/<INDEX>`. +
+   Expected: The loan is returned successfully. The display message details that the loan has been deleted, and gives a suggestion for the cost price of the duration of the loan.
+
+.. Choose another ongoing loan from the list and run `return i/<INDEX>`. +
+   Expected: The loan is returned successfully as well. The display message details that the loan has been deleted, and gives a suggestion for the cost price of the duration of the loan.
+
+. Negative tests:
+.. Test case: `return i/1` +
+   Expected: First loan from the list is marked as returned. Details of the returned loan shown in the status message. Timestamp in the status bar is updated. The values of the card on the left of the application has been changed to reflect the returning of the loan.
+
+.. Test case: `return i/0` +
+   Expected: No loan is returned. Error details shown in the status message. Status bar remains the same.
+.. Other incorrect `return` commands to try:
+... `return`
+... `return x` (where `x` is larger than the list size)
+... `return i/aaa` (Where `aaa` is a string)
+... `return y` (where `y` is a loan that has already returned). +
```

Comment:
"`y` is the index of a loan that has already been returned."
If you're planning to add more loan statuses, it may be helpful to rephrase this as "`y` is the index of a loan that is not `Ongoing`."